js/js.tests/test/org/jetbrains/kotlin/js/test/InteropJsTestChecker.kt

```diff
@@ -5,65 +5,61 @@
 
 package org.jetbrains.kotlin.js.test
 
-import jdk.nashorn.api.scripting.NashornScriptEngineFactory
-import jdk.nashorn.api.scripting.ScriptObjectMirror
-import jdk.nashorn.internal.runtime.ScriptRuntime
+import org.jetbrains.kotlin.js.test.interop.GlobalRuntimeContext
+import org.jetbrains.kotlin.js.test.interop.InteropEngine
+import org.jetbrains.kotlin.js.test.interop.InteropNashorn
+import org.jetbrains.kotlin.js.test.interop.InteropV8
 import org.junit.Assert
-import javax.script.Invocable
-import javax.script.ScriptEngine
 
-fun createScriptEngine(): ScriptEngine =
-    // TODO use "-strict"
-    NashornScriptEngineFactory().getScriptEngine("--language=es5", "--no-java", "--no-syntax-extensions")
+private val USE_J2V8_INTEROP_FOR_JS_TESTS = java.lang.Boolean.getBoolean("org.jetbrains.kotlin.use.j2v8.interop.for.js.tests")
```

Comment:
Could you please move this flag into `KotlinTestUtil.java`? I think keeping such flags in one specific place is good.

Reply:
@romanart in that case I'll  have to make it (unlike other flags declared there) public or come up with some contrived  way to pass it to InteropJsTestChecker (who we'll call it btw - I mean ScriptEngineTestChecker sounds slightly off).

If we are ok with this - sure - let me do it.

Reply:
It is also possible to pass engine type as parameter from outside

Reply:
My point is to keep all flags in one place